### DISCUSSION TITLE
Feature Request: Pie chart of languages used in repositories created/contributed

### DISCUSSION THREAD

**Discussion by IslandRhythms** [community/community#40647] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I don't know if this is the right place for this but I am requesting a feature on the profile page that would show a pie chart or some other visualizer of languages used in created repositories, as well as repositores the user has contributed to.

**Comment by ossanna16** [comment]
Hi there @IslandRhythms and welcome to our community! Thank you for the feedback!
